CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # FL Accelerator Changelog
 
+##  October 2022 release
+
+We are excited to announce the release of the September iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
+
+Here are the new features.
+
+
+### Repository structure
+- Enabled CI/CD on the repo to ensure nothing breaks.
+
+### Provisioning
+- Decoupled the provisioning of the workspace, the orchestrator, and the silos.
+- New scripts for provisioning various kinds of resources, all listed in the [cookbook](./docs/provisioning/README.md) (orchestrator behind vnet, silos behind vnets, silos using an AKS cluster with confidential compute, silos using an existing storage account).
```

Comment:
I'd put this first, and sell it better, like:

- A new provisioning "cookbook" so that you can pick and choose your own setup based on provided templates (orchestrator behind vnet, ...)